e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -79,8 +121,10 @@
       "properties.desired": {
         "schemaVersion": "1.0",
         "routes": {
-          "upstreamLoadGen1": "FROM /messages/modules/loadGen1/outputs/output1 INTO $upstream",
-          "upstreamLoadGen2": "FROM /messages/modules/loadGen2/outputs/output1 INTO $upstream",
+          "loadGen1ToRelayer1": "FROM /messages/modules/loadGen1/outputs/output1 INTO BrokeredEndpoint(\"/modules/Relayer/inputs/input1\")",
+          "relayer1ToCloud": "FROM /messages/modules/Relayer/output1 INTO $upstream",
+          "loadGen2ToRelayer2": "FROM /messages/modules/loadGen2/outputs/output2 INTO BrokeredEndpoint(\"/modules/Relayer/inputs/input2\")",
+          "relayer2ToCloud": "FROM /messages/modules/Relayer/output2 INTO $upstream"
```

Comment:
I think you need to have 2 relay modules; otherwise it doesn't distinguish the message is received from which module based on current implementation of Relayer module.